server/migrations/20171226132000-create-todo.js

```diff
@@ -0,0 +1,33 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Todos', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      title: {
+        type: Sequelize.STRING,
+      },
+      complete: {
+        type: Sequelize.BOOLEAN,
+      },
+      UserId: {
```

Comment:
could you make this `camelCase`?

Reply:
which one should I make camelCase? the migration file name? I did not name that myself. I hope it won't cause problem if I change the name

Reply:
`userId`

Reply:
The comments are most times directly below the line affected.